### DISCUSSION TITLE
3DS stuck on GM9 screen

### DISCUSSION THREAD

**Discussion by webupnz** [d0k3/GodMode9#806] [Q&A]
our 3DS has recently become stuck on GM9 screen and no matter what i do it wont progress into the normal 3DS homescreen

Randomly sometimes it works then most of the time it doesnt

![20230414_141347](https://user-images.githubusercontent.com/97335031/231925145-c3a1025d-f67c-40d7-917f-7a8d5bd0585c.jpg)
![20230414_141345](https://user-images.githubusercontent.com/97335031/231925170-c840b77b-38fc-4d31-a326-2b6a109b7733.jpg)
![20230414_141400](https://user-images.githubusercontent.com/97335031/231925156-7a28b64c-9f5b-4643-833a-ec5ba057759f.jpg)
![20230414_141404](https://user-images.githubusercontent.com/97335031/231925161-227e6e60-7060-4250-b49c-f6c4e97b2946.jpg)
![20230414_141412](https://user-images.githubusercontent.com/97335031/231925163-b58a116f-b955-4fde-8fac-3045f11a3228.jpg)
![20230414_141415](https://user-images.githubusercontent.com/97335031/231925166-9475a23c-81aa-466c-aec5-94f03fb32a65.jpg)

**Comment by Epicpkmn11** [comment]
Do you know if you have Fastboot3DS or boot9strap? You can check by holding <kbd>HOME</kbd> while turning on the console, if Fastboot3DS it'll open a different menu, boot9strap it won't change anything.

If Fastboot3DS: Check your config and make sure GodMode9 isn't set as the default option somehow, maybe switch which button it's mapped to, if any.

If boot9strap: Maybe your <kbd>START</kbd> button has gotten stuck down somehow, you could try deleting `sd:/luma/payloads/godmode9.firm` for now and see if that helps, then see if your <kbd>START</kbd> button is working right
